### ISSUE TITLE
Multiple render of cards when route is replaced

### ISSUE THREAD

**Issue by georgberecz** [LeoLeBras/react-router-navigation#23]
Hey guys,
I checked out your package and came across some problems on iOS and Android. It seems that cards are rendered multiple times by the `CardStack` even though they should only be rendered once when I replace the current history.
I dispatch a `replace` from react-router-redux in an action creator that reroutes to my index Card like this:
```
export function setStoryActive(userId, storyId) {
  return (dispatch) => {
    dispatch(setStoryActiveStart());
    return postActiveStory(userId, storyId)
      .then(json => {
        dispatch(setStoryActiveSuccess(json));
        dispatch(getStoryProgress(userId, storyId))
        dispatch(replace('/'));
      }).catch((e) => {
        console.error(e);
        dispatch(setStoryActiveError(e.message));
      })
  };
}
```
In general, the redirect and replace works. However, it seems that despite the replace the old card is still rendered below the new replaced one:
![image](https://cloud.githubusercontent.com/assets/7314829/26771639/9a1e9128-4974-11e7-89de-88f87200d0d3.png)
As can be seen that leads to the overlapped text in the middle and the rendering of the back button (which should not be present and which actually cannot be clicked, because it lays beneath).

I'm using a setup with
```   
    "react-router": "^4.1.1",
    "react-router-native": "^4.1.1",
    "react-router-navigation": "^1.0.0-rc.0",
    "react-router-redux": "^5.0.0-alpha.6",
```
I would really appreciate fast help with that problem. :)

Btw, I love your package! Saw the great lightning talk from @CharlesMangwa at ReactConf and really like the way you guys handle the navigation! 👍

**Comment by georgberecz** [comment]
Apparently, while I was writing the issue somebody else came across the exact same problem: https://github.com/LeoLeBras/react-router-navigation/issues/22

**Comment by LeoLeBras** [comment]
`rc` release was not published because there are some bugs to fix. I'm currently busy, but I think I'll release the `rc` next week with this issue fixed.

**Comment by LeoLeBras** [comment]
This should be fixed in `1.0.0-rc.2`
